provider/win_driver_utils.py

```diff
@@ -194,6 +194,71 @@ def copy_file_to_samepath(session, test, params):
                        " the detailed info:\n%s." % output)
 
 
+def enable_driver(vm, test, params, device_id):
```

Comment:
1. Let's avoid pass the whole params when there is only one of them been used.
2. I would recommend include the `if...else...` condition here inside this function,
```
def enable_driver(session, test, enable_method='install', **kwargs):
    if enable_method == 'install':
        return install_driver_by_virtio_media(session, test, **kwargs)
    ...
```
And if you need you might check avocado-vt/virttest/qemu_monitor.py to find examples for how to define and utilize `**kwargs`.